### PR TITLE
[IR-3020] studio: hide ScenePreviewCameraComponent

### PR DESCRIPTION
**Pull request by dtlehrer**
## Summary
[IR-3020]

## References
closes #_insert number here_

## QA Steps

[IR-3020]: https://tsu.atlassian.net/browse/IR-3020?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by HexaField — CHANGES_REQUESTED** [review]
we need to use feature flags for this

File: packages/engine/src/FeatureFlagsState.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const FeatureFlagsState = defineState({
   initial: {} as Record<string, boolean>,
   enabled(flagName: string) {
     const state = getMutableState(FeatureFlagsState)[flagName].value
-    return typeof state === 'boolean' ? state : true
+    return typeof state === 'boolean' ? state : false // returns false if state has not been set
```
Comment (HexaField) [inline packages/engine/src/FeatureFlagsState.tsx]: we want to preserve the default functionality of everything being enabled for local dev. we either need to seed everything into the database, or return this to default to true.

**Review by HexaField — DISMISSED** [review]

File: packages/engine/src/FeatureFlagsState.tsx (hunk 1 above)
Comment (dtlehrer) [reply packages/engine/src/FeatureFlagsState.tsx]: makes sense for local dev at the moment (and would be nice to have optionality around what is on/off + the ability to quickly toggle or set flag values in the future)

is there somewhere we are turning flags off for features that we don't want to be enabled in other environments?

Comment (dtlehrer) [reply packages/engine/src/FeatureFlagsState.tsx]: or should we just go ahead and seed/store key-value pairs for each flag in the db per environment?

noticing that we're reaching a point where there are several parts of the studio that others are asking to hide/disable beyond the local env

Comment (HexaField) [reply packages/engine/src/FeatureFlagsState.tsx]: reverting is good for now, we can address seeding later

**Review by MichaelEstes — APPROVED** [review]

**Pull request merged**